core/src/Kokkos_Tuners.hpp

```diff
@@ -256,13 +256,13 @@ auto get_point_helper(const PointType& in, const ArrayType& indices,
 template <typename PointType, typename ArrayType>
 struct GetPoint;
 
-template <typename PointType, size_t X>
+template <typename PointType, size_t SZ>
```

Comment:
I'd use something more readable/verbose.
```suggestion
template <typename PointType, size_t ArraySize>
```